marge/app.py

```diff
@@ -213,6 +213,17 @@ def regexp(str_regex):
         action='store_true',
         help='Disable fast forwarding when merging MR batches'
     )
+    parser.add_argument(
+        '--use-merge-commit-batches',
+        action='store_true',
+        help='Use merge commit when creating batches, so that the commits in the batch MR '
+             'will be the same with in individual MRs. Please add sudo scope to the access token.\n',
```

Comment:
If the `sudo` scope is needed for the functionality, we should change the wording used here. "Please add" implies it's nice to have, but not necessary. Maybe something like `Requires sudo scope in the access token.` ?